wikidp/static/js/wikidp.js

```diff
@@ -10,6 +10,7 @@ var oauthController = {
       success: function (data, textStatus, jqXHR) {
         console.log(data)
         if (data.auth === false) {
+          sessionStorage.setItem('returnTo', window.location.pathname)
```

Comment:
super small nit: When playing around with the redirect, I noticed it doesn't preserve the query params. Perhaps we can add that here. Super minor because query params are mainly used for deep-linking with search results but just in case we start adding more useful metadata in there it'd be nice to have

Reply:
Fixed in latest version